test/module/shared_model/builders/protobuf/transport_builder_test.cpp

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2018 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "builders/protobuf/transport_builder.hpp"
+#include <gtest/gtest.h>
+#include "block.pb.h"
+#include "test_block_builder.hpp"
+#include "test_proposal_builder.hpp"
+#include "test_query_builder.hpp"
+#include "test_transaction_builder.hpp"
+
+using namespace shared_model;
+using namespace iroha::expected;
+
+class TransportBuilderTest : public ::testing::Test {
+ public:
+  void SetUp() override {
+    created_time = iroha::time::now();
+    account_id = "account@domain";
+    quorum = 2;
+    counter = 1048576;
+    hash = std::string(32, '0');
+    height = 1;
+  }
+
+  Transaction createTransaction() {
+    TestTransactionBuilder builder;
+    auto tx = builder.createdTime(created_time)
+                  .txCounter(counter)
+                  .creatorAccountId(account_id)
+                  .setAccountQuorum(account_id, quorum)
+                  .build();
+    return tx;
+  }
+
+  Query createQuery() {
+    TestQueryBuilder builder;
+
+    auto query = builder.createdTime(created_time)
+                     .creatorAccountId(account_id)
+                     .getAccount(account_id)
+                     .queryCounter(counter)
+                     .build();
+    return query;
+  }
+
+  Block createBlock() {
+    TestBlockBuilder builder;
+
+    auto block =
+        builder.transactions(std::vector<Transaction>({createTransaction()}))
+            .txNumber(1)
+            .height(1)
+            .prevHash(crypto::Hash("asd"))
+            .createdTime(created_time)
+            .build();
+    return block;
+  }
+
+  Proposal createProposal() {
+    TestProposalBuilder builder;
+
+    auto proposal =
+        builder.transactions(std::vector<Transaction>({createTransaction()}))
+            .height(1)
+            .createdTime(created_time)
+            .build();
+
+    return proposal;
+  }
+
+ protected:
```

Comment:
What is the point to make fields protected?

Reply:
Simply the style. Minor issue